sql/src/main/java/io/crate/exceptions/LicenseViolationException.java

```diff
@@ -24,13 +24,17 @@
 
 import java.util.Locale;
 
-public class ExpiredLicenseException extends ValidationException implements ClusterScopeException {
+public class LicenseViolationException extends ValidationException implements ClusterScopeException {
 
-    private static final String MESSAGE_TEMPLATE = "License is expired - %s. " +
-                                                   "Please request a new license and " +
-                                                   "use 'SET LICENSE' statement to register it";
+    private static final String MESSAGE_TEMPLATE = "License is violated - %s. " +
```

Comment:
cc: @lukas-bernert is the wording here okay with you?

Reply:
Yes.
As long as we include the link to crate.io/license-update in the cluster checks?